src/components/Coaching/CoachingRow/CoachingRow.tsx

```diff
@@ -61,31 +66,50 @@ export const CoachingRow: React.FC<Props> = ({
     },
   });
 
+  const usersList = users.nodes
+    .map((user) => user.firstName + ' ' + user.lastName)
+    .join(', ');
+
   return (
     <>
       <CoachingRowWrapper role="listitem">
-        <CoachingNameText variant="h6" color="primary">
-          <NextLink
-            href={{
-              pathname: '/accountLists/[accountListId]/coaching/[coachingId]',
-              query: { accountListId: accountListId, coachingId: id },
-            }}
-            passHref
-          >
-            <Link flex={1} underline="hover">
-              {name}
-            </Link>
-          </NextLink>
-          <Button
-            onClick={(event) => {
-              event.preventDefault();
-              setConfirmingDelete(true);
-            }}
-            aria-label={t('Remove Access')}
-          >
-            <VisibilityOff />
-          </Button>
-        </CoachingNameText>
+        <Box display="flex">
+          <Box flex={1}>
+            <CoachingNameText variant="h6" color="primary">
+              <NextLink
```

Comment:
None of the `CoachingNameText` styles seem like they change anything. It already has `margin: 0`, it's `cursor: pointer` because it's a link, and `display: inline` doesn't change anything either. Unless I'm missing something, I'd just make this a `Typography` and remove the `CoachingNameText` styled component and the `color="primary"` attribute since the coloring comes from the nested link.

Reply:
I like having display='inline' because it makes the hover only on the actual link text rather than the hover applying to the whole empty line space all the way across the page.

Reply:
That makes sense. I missed that.